suite-common/wallet-graph/src/graphUtils.ts

```diff
@@ -759,6 +772,17 @@ export const fetchAccountBalanceHistoryWithBalanceBefore = async (
         account,
         response.payload,
     );
+
+    const { symbol } = account;
+
+    // cannot find ticker config for symbol, e.g. BTC TEST - fallback to 0
+    if (!getTickerConfig({ symbol }) && zeroFiatRatesFallback) {
+        return balanceHistory.map(balanceHistoryPoint => ({
+            ...balanceHistoryPoint,
+            rates: zeroFiatRatesFallback,
+        }));
```

Comment:
Q: Why use D.map above and not here? Is there a pattern or why did you not use it here?

Reply:
This is mapping over array